src/pages/leeseyoon/Login/Login.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./Login.scss";
+import "../styles/common.scss";
+
+const Login = () => {
+  const navigate = useNavigate();
+  const goToMain = () => navigate("/Mainleeseyoon");
+
+  const [userId, setUserId] = useState("");
+  const [userPw, setUserPw] = useState("");
+  const [idValid, setIdValid] = useState(false);
+  const [pwValid, setPwValid] = useState(false);
+
+  const handleId = e => {
+    setUserId(e.target.value);
+    setIdValid(e.target.value.includes("@") ? true : false);
+  };
+  const handlePw = e => {
+    setUserPw(e.target.value);
+    setPwValid(e.target.value.length >= 5 ? true : false);
+  };
```

Comment:
idValid, pwValid는 userId, userPw state로 계산해서 관리할 수 있음.
```suggestion
  const isIdValid = userId.includes('@');
  const isPwValid = userPw.length >= 5;
```